install.sh

```diff
@@ -194,94 +195,95 @@ __install_installer_deps() {
 	done
 }
 
-__install_bro() {
+__install_zeek() {
 
-
-	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" -o "$_OS_CODENAME" == "trusty" ] ; then
-		# Bro 2.6 will not compile under Debian 8/ Ubuntu Trusty/ Xenial in the OpenSuse Build Service
-		# Install Bro 2.5.5
-		__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro/xUbuntu_$(lsb_release -rs)/ /" \
-			"Bro" \
-			"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro/xUbuntu_$(lsb_release -rs)/Release.key"
+	if  [ "$_OS" == "Ubuntu" ] && [ "$_OS_CODENAME" == "xenial" ] ; then
+		# The Zeek Project does not host a package repo for Ubuntu 16.04.
+		# Use Security Onion's version of Zeek.
+		sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 9373E47F9BF8216D23D32BBBE1E6759023F386C7
+		__add_deb_repo "deb http://ppa.launchpad.net/securityonion/stable/ubuntu xenial main" \
+		"securityonion-ubuntu-stable-xenial"
+		__install_packages securityonion-bro
+		ln -s /opt/bro /opt/zeek
 	else
 		case "$_OS" in
 			Ubuntu)
-				__add_deb_repo "deb http://download.opensuse.org/repositories/home:/logan_bhis:/branches:/network:/bro-2-6/xUbuntu_$(lsb_release -rs)/ /" \
-					"Bro" \
-					"https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/xUbuntu_$(lsb_release -rs)/Release.key"
+				__add_deb_repo "deb [ arch=$(dpkg --print-architecture) ] http://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/ /" \
+				"security:zeek" \
+				"https://download.opensuse.org/repositories/security:/zeek/xUbuntu_$(lsb_release -rs)/Release.key"
 				;;
 			CentOS|RedHatEnterprise|RedHatEnterpriseServer)
-				__add_rpm_repo https://download.opensuse.org/repositories/home:logan_bhis:branches:network:bro-2-6/CentOS_7/home:logan_bhis:branches:network:bro-2-6.repo
+				__add_rpm_repo "https://download.opensuse.org/repositories/security:zeek/CentOS_7/security:zeek.repo"
 				;;
 		esac
+		__install_packages zeek-lts
 	fi
 
-	__install_packages bro broctl
 
-	if [ -d /opt/bro/logs/ ]; then		#Standard directory for Bro logs when installed by Rita
-		chmod 2755 /opt/bro/logs
-	elif [ -d /var/log/bro/ ]; then		#Standard directory for Bro logs when installed by apt...
-		mkdir -p /opt/bro/logs/		#...and we move the log storage over to /opt/bro/logs so we can have one place to mount external storage.
-		chmod 2755 /opt/bro/logs
-		mv -f /var/log/bro /var/log/bro.orig
+	if [ -d /opt/zeek/logs/ ]; then		#Standard directory for Zeek logs when installed by Rita
+		chmod 2755 /opt/zeek/logs
+	elif [ -d /var/log/zeek/ ]; then		#Standard directory for Zeek logs when installed by apt...
```

Comment:
We should check what directory the `securityonion-bro` package uses. In the distro I think it's `/nsm/zeek/logs` or something close to that.

Reply:
The newest version of SO is using `/opt/zeek/ -> /opt/bro`. I think I've seen source installation guides using `/nsm` but I'll double check.

Reply:
On the 16.04 system I used this installer on I see that logs go to `/opt/bro/logs`. `/nsm/bro/logs` exists but it seems to not be used as my logs aren't going there. And `/opt/zeek` is a symlink to `/opt/bro` like you said. I checked and `[ -d /opt/zeek/logs ]` works as expected. I think we should be good to go.